sql-6b.sql

```diff
@@ -0,0 +1,148 @@
+
```

Comment:
NAMING INCONSISTENCIES
 Students table has "customer" prefixed columns instead of "student"
 Inconsistent casing
 Some tables use singular form (quiz) while others use plural (assignments)
 
 
 DATA LENGTH/TYPE ISSUES

Email lengths (VARCHAR(25)) too short - should be at least VARCHAR(100)
Names length (VARCHAR(25)) might be insufficient - consider VARCHAR(50)
Course name VARCHAR(25) too short for descriptive course titles
Remarks VARCHAR(100) might be too short for detailed feedback
Address field too generic - should be split into components

MISSING NOT NULL CONSTRAINTS
MISSING CHECK CONSTRAINTS